### DISCUSSION TITLE
Use a popup window instead of site integration?

### DISCUSSION THREAD

**Discussion by dikkadev** [jolution/le-checkout-jira#54] [Q&A]
Has the possibility of using a popup window instead of injecting elements into the site been evaluated?

IMO, this would be especially useful for better compatibility (regarding UX and development effort) when other sites will be supported.

**Comment by juliankasimir** [comment]
Hi @Sett17,

Thank you for your feedback! I think it's a great idea. It would also allow us to use our own design in the chrome extension popup. However, we also like having the option to work directly on the ticket with just one click. So, we're considering offering both options, where users could choose via a checkbox whether they want to work in Popup mode or in an integrated way.

Best regards,
Julian

**Reply by dikkadev** [reply]
Making both available is a good idea. I'd imagine having the possibility of the popup (or sidebar, or ...) all the time and enabling you to have the injected elements on sites where that is implemented.

That would also make it easier to ship for different websites faster, as I'd imagine the injected elements are a huge task. Completion of that can then be offset from supporting the site in general.
